client/core/bookie.go

```diff
@@ -34,7 +34,7 @@ type BookFeed struct {
 // feed when it's no longer being used.
 func NewBookFeed(close func(feed *BookFeed)) *BookFeed {
 	return &BookFeed{
-		C:     make(chan *BookUpdate, 1),
+		C:     make(chan *BookUpdate, 256),
```

Comment:
The problem I'm trying to avoid is at https://github.com/decred/dcrdex/blob/9efbc84fefa42d80129d0978fb7a43d7f72314bd/client/core/bookie.go#L128-L133

but it raises the question of whether that's the right pattern.

Reply:
I was battling with a similar challenge in https://github.com/decred/dcrdex/pull/287 with sends on the `orderRouter` channel.  Ideally the channel buffer should only be a performance consideration (at least beyond a buffer of 1), and things should work if the buffer fills up.  It's tough to be sure, but in the market's case it was a matter of proper synchronization between the sender and receiver so that the sender can guarantee there's a receiver on send.  I'll give a proper review of this PR  asap.